Integrations/Digital_Defense_FrontlineVM/Digital_Defense_FrontlineVM.py

```diff
@@ -0,0 +1,806 @@
+    vuln_stat_output['Vulnerabilities'] = len(vuln_list)
+    vuln_data_list = []
+
+    # Condensing Vulns for HumanReadable and EntryContext:
+    for vuln in vuln_list:
+        vuln_obj = create_vuln_obj(vuln)
+        vuln_severity = str(vuln['severities']['ddi']).capitalize()
+        if vuln_stat_output.get(vuln_severity):
+            vuln_stat_output[vuln_severity] += 1
+        else:
+            vuln_stat_output[vuln_severity] = 1
+        vuln_data_list.append(vuln_obj)
+
+    return {
+        'data_output': vuln_data_list,    # condensed vuln data pulled from Frontline.Cloud
+        'stat_output': vuln_stat_output,    # statistical vulnerability data
+    }
+
+
+def get_host_id_from_ip_address(ip_address):
+    '''
+        Get host ID within Frontline.Cloud given IP address.
+        Host ID used to pull vulnerability data for that specific host.
+    '''
+    hosts_with_given_ip = get_fvm_data(HOST_ENDPOINT, params={'_0_eq_host_ip_address': str(ip_address)})
+    if len(hosts_with_given_ip) < 1:
+        msg = 'Host not found within Frontline.Cloud given host IP Address.'
+        demisto.debug('Frontline.Cloud get_host_id_from_ip_address -- ' + msg)
+        return_error("Error: " + msg)
+    first_relevant_host = hosts_with_given_ip[0]
+    return first_relevant_host.get('id')
+
+
+def get_vulns_command():
+    ''' Pulls vulnerability information from Frontline.Cloud    '''
+    # Get Arugments:
+    severity = demisto.args().get('severity')
+    min_severity = demisto.args().get('min_severity')
+    max_days_since_created = demisto.args().get('max_days_since_created')
+    min_days_since_created = demisto.args().get('min_days_since_created')
+    host_id = demisto.args().get('host_id')
+    ip_address = demisto.args().get('ip_address')
+
+    if ip_address:
+        host_id = get_host_id_from_ip_address(ip_address)
+    vulns = get_vulns(severity, min_severity, max_days_since_created, min_days_since_created, host_id)
+
+    # get both vuln data and vuln statistical output
+    output = get_vuln_outputs(vulns)
+
+    # Vuln Data Output:
+    vuln_data_table_name = "FrontlineVM: Vulnerabilities Found"
+    vuln_data_output = output.get('data_output')
+
+    # Vuln Statistical Output:
+    vuln_stat_table_name = "FrontlineVM: Vulnerability Statisctics"
+    vuln_stat_output = output.get('stat_output')
+    vuln_stat_headers = list(vuln_stat_output.keys())
+
+    demisto.results([
+        {
+            'Type': entryTypes['note'],
+            'Contents': vulns,
+            'ContentsFormat': formats['json'],
+            'HumanReadable': tableToMarkdown(vuln_data_table_name,
+                                             vuln_data_output,
+                                             headers=VULN_DATA_HEADERS,
+                                             removeNull=True),
+            'ReadableContentsFormat': formats['markdown'],
+            'EntryContext': {'FrontlineVM.Vulns(val.ID && val.ID == obj.ID)': vuln_data_output}
```

Comment:
let's change ID here to "vuln-id", since Vulns don't have an ID context path

Reply:
Updated `'EntryContext': {'FrontlineVM.Vulns(val.ID && val.ID == obj.ID)': vuln_data_output}`

to `'EntryContext': {'FrontlineVM.Vulns(val.vuln-id && val.vuln-id == obj.vuln-id)': vuln_data_output}`